src/YAMLValidator/InputProcessor.cs

```diff
@@ -0,0 +1,112 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.CommandLine;
+
+namespace Microsoft.PowerPlatform.PowerApps.Persistence;
```

Comment:
Please add "YAMLValidator" so it becomes Microsoft.PowerPlatform.PowerApps.Persistence.YAMLValidator;